### PR TITLE
Alexa build

### PR DESCRIPTION
**Pull request by muethela**
Building 20201124A host, adding 2MASS transmission curves, and updating how photometry tables are merged

### CODE REVIEW

File: frb/data/Galaxies/Literature/fong2021_photom.ascii [hunk 1]
```diff
@@ -0,0 +1,2 @@
+|     Name |         ra |        dec | 2MASS_J | 2MASS_J_err | 2MASS_H | 2MASS_H_err | 2MASS_K | 2MASS_K_err |
```
Comment (profxj) [inline frb/data/Galaxies/Literature/fong2021_photom.ascii:1]: Just confirming -- these are *without* Galactic extinction corrections, right?

File: frb/data/analysis/CIGALE/2MASS_H.dat [hunk 2]
```diff
@@ -0,0 +1,175 @@
+13000.0 0.0
```
Comment (profxj) [inline frb/data/analysis/CIGALE/2MASS_H.dat:1]: Can you add a header to each of these similar
to what we have for the others?  I guess it is only
for `CIGALE` which we might not use much anymore,
but good for uniformity.

Here is the header for `GMOS_N_r` by way of example:

```
#GMOS_N_r
#energy
#gemini Gmos N r band http://svo2.cab.inta-csic.es/theory/fps/index.php?id=Gemin
i/GMOS-N.r&&mode=browse&gname=Gemini&gname2=GMOS-N#filter
```

**Review by profxj — CHANGES_REQUESTED** [review]
Just a couple of minor things.

Thanks for this!

File: frb/data/Galaxies/Literature/fong2021_photom.ascii (hunk 1 above)
Comment (muethela) [reply frb/data/Galaxies/Literature/fong2021_photom.ascii:1]: Correct, these are not corrected for extinction

File: frb/data/analysis/CIGALE/2MASS_H.dat (hunk 2 above)
Comment (muethela) [reply frb/data/analysis/CIGALE/2MASS_H.dat:4]: Added

**Review by profxj — APPROVED** [review]
All good!

**Pull request merged**